lib/update.js

```diff
@@ -1,9 +1,33 @@
 'use strict';
 
+var async = require('async');
+var request = require('supertest');
+
 module.exports = function updateAccount(serviceData, cursor, queues, cb) {
-  // Update documents from provider
-  // You may define this as an helper function
-  // this function is pinged on update, with the data stored by retrieveTokens()
-  // You can do queues.worker_name.push(task)
-  cb(null, new Date());
+
+  var url = '/v1/people/~/connections?oauth2_access_token=' + serviceData.access_token + '&format=json';
+
+  if(cursor !== null) {
+    url += '&modified=new&modified-since=' + cursor;
+  }
+  cursor = new Date().getTime();
+
+  async.waterfall([
+   function getConnections(cb) {
+    request('https://api.linkedin.com')
+    .get(url)
+    .expect(200)
+    .end(cb);
+  },
+  function parseConnections(res, cb) {
+    if(res.body.values && res.body.values.length > 0) {
```

Comment:
you don't need to check for the length, in the worst case the forEach will iterate on empty